js/main.js

```diff
@@ -1,18 +1,16 @@
-var $pokemonImg = document.querySelector('.pokemon-img')
+var rightAnswer = false;
+var currentID = 0;
+var randomIDList = shuffle(1, 10);
```

Comment:
Where is the `10` coming from here? Could you use the `length` of the source of your data instead?